packages/ui/components/friend-row/index.tsx

```diff
@@ -18,7 +18,7 @@ import { calculateBalance, convertCurrency, getPrimaryCurrency, getPendingFromFr
 import { connect } from 'react-redux'
 
 import language from 'language'
-const { debtManagement } = language
+const { debtManagement, pendingTransactionsLanguage } = language
```

Comment:
I think the word `Pending` could also be the words `In Progress`, but `Pending` is fine.
This is also another button that asks "is there a good icon for me"?